torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -638,6 +638,31 @@ def rewrite_node(node):
                     input = g.call_function(ttnn.to_layout, args=(input, TtnnRowMajorLayout()))
                 return g.call_function(ttnn.pad, args=(input, full_pad, value))
 
+            # NOTE(jdh8): We only convert max_pool2d without indices
+            if node.target == torch.ops.aten.max_pool2d_with_indices.default:
+                try:
+                    (user,) = node.users.keys()
+                    assert user.target.__name__ == "getitem"
+                    assert user.args == (node, 0)
+                except:
+                    return None
+
+                in_n, in_c, in_h, in_w = args[0].meta["val"].size()
+                out_n, out_c, out_h, out_w = node.meta["val"][0].size()
+                input_nhwc = g.call_function(ttnn.permute, (args[0], (0, 2, 3, 1)))
+                input_nhwc = g.call_function(ttnn.reshape, (input_nhwc, (-1, in_c)))
```

Comment:
And then, this line reshapes input activation to (1\*64\*64, 3)

Reply:
I think conv expects [1, 1, 64*64, 3], but in bfloat having 3 as last dimension won't work in Row Major, because Page Size is 32 bit, so we need more elements. 
You can check it if in your test you will have 16 instead of 3. 
Can you please try it?